lib/eslint-patch.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+var meld = require('meld');
+var docs = require('./docs');
+var Config = require("eslint/lib/config");
+var ConfigUpgrader = require('./config_upgrader');
+
+var supportedPlugins = ['react', 'babel'];
+
+module.exports = function patcher(eslint) {
+
+  meld.around(eslint.CLIEngine, 'loadPlugins', function(joinPoint) {
```

Comment:
This was likely added when we supported ESLint 2 on the way to supporting ESLint 3. Is there a similar function in ESLint 3 that we can meld?